models/navigation/fact_learner_last_course_visit.sql

```diff
@@ -0,0 +1,15 @@
+{{
+    config(
+        materialized="materialized_view",
+        schema=env_var("ASPECTS_XAPI_DATABASE", "xapi"),
+        engine=get_engine("ReplacingMergeTree()"),
+        primary_key="(org, course_key, actor_id)",
+        order_by="(org, course_key, actor_id)",
+        partition_by="(toYYYYMM(last_visited))",
+        ttl=env_var("ASPECTS_DATA_TTL_EXPRESSION", ""),
+    )
+}}
+
+select org, course_key, actor_id, max(emission_time) as last_visited
+from {{ ref("navigation_events") }}
+group by org, course_key, actor_id
```

Comment:
`ASPECTS_DATA_TTL_EXPRESSION` uses `emission_time`. You can update the upstream reports to rename the column